ui/pages/confirmations/components/confirm/info/approve/edit-spending-cap-modal/edit-spending-cap-modal.tsx

```diff
@@ -124,6 +124,8 @@ export const EditSpendingCapModal = ({
     decimals &&
     parseInt(decimals, 10) < countDecimalDigits(customSpendingCapInputValue);
 
+  const showSpecialCharacterError = /[-+e]/u.test(customSpendingCapInputValue);
```

Comment:
Minor, is it worth a constant for the regex such as `SPECIAL_CHARACTERS_PATTERN`?